modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/upgrade/v2_0_0/SXPBlueprintUpgradeProcess.java

```diff
@@ -35,6 +35,7 @@ public class SXPBlueprintUpgradeProcess extends UpgradeProcess {
 	protected void doUpgrade() throws Exception {
 		_upgradeSearchBarPortlets();
 		_upgradeSXPBlueprintOptionsPortlets();
+		_upgradeLLSOOptionsPortlets();
```

Comment:
@felipelorenz I'm not sure, but if the abbreviation of LowLevelSearchOptionsPortlet is LLSO so the method name should be `_upgradeLowLevelSearchOptionsPortlets()` or `_upgradeLLSO()`,

I prefer the first one, it's more accurate than LLSO.

another point is that this upgrade needs to come before `_upgradeSearchBarPortlets` to maintain alphabetical order.